mantidproject/mslice
# Implemenation of automatic workspace selection after Load and Calculate Projection.

This PR changes the GUI behaviour to set the selected workspace(s) in the workspace manager to be the result of the `Load` or `Calculate Projection` actions after they complete.

**To test:**

Run the GUI, and load a data file. On success check that the loaded workspace is selected in the workspace manager. Now click `Calculate Projection` - check the created `_QE` or `_EQ` workspace is now selected. (Also try it with multiple selection for `Calculate Projection` and check that all the created workspaces are selected).

Fixes #96.

Unit tests to follow...
